apps/hash-external-services/package.json

```diff
@@ -7,6 +7,7 @@
     "codegen": "touch ../../.env.local",
     "deploy": "docker compose --project-name hash-external-services --file docker-compose.yml --file docker-compose.type-fetcher.yml --env-file ../../.env --env-file ../../.env.local",
     "deploy:offline": "docker compose --project-name hash-external-services --file docker-compose.yml --env-file ../../.env --env-file ../../.env.local",
+    "deploy:prod": "docker compose --project-name hash-external-services --file docker-compose.prod.yml --env-file ../../.env --env-file ../../.env.local",
```

Comment:
I think this is fine, but one small detail is that we're layering the `env-file`s which forces the user to have a newer docker compose version. I know we set a repo-wide minimum version, but I'm thinking for hte production deployment script we should perhahps have a `.env.prod` that is an exact env file that can be used on its own? cc @indietyp

Reply:
I was thinking about a `.env.prod` (and `.env.test`) as well but I'd like us not to duplicate every entry as this will be a pretty big maintenance overhead and they will fall out of sync. Similarly, I also like to make the `docker-compose.prod.yml` layered, so the maintenance cost there will be minimal as well.

Reply:
We currently have docker compose `2.16.0`/`2.12.0` deployed. I'd need to schedule a downtime, but that wouldn't pose a problem

Reply:
We currently have docker compose `2.16.0`/`2.12.0` deployed. I'd need to schedule a downtime, but that wouldn't pose a problem